nservicebus/licensing/index_validity_core_[3,4.5).partial.md

```diff
@@ -0,0 +1 @@
+The license is valid if the `LicenseVersion` attribute in the license data is greater than or equal to the `{major}.{minor}` part of the fileversion for the NServiceBus assembly used by the endpoint.
```

Comment:
@andreasohlund unfortunately partials can only match what the version switch will allow. in this case i recommend u have  one partial targeting "5 and lower" with version specific language in it. and then one for "6 and higher"